### DISCUSSION TITLE
running Tasmota on ESP8685-WROOM-04

### DISCUSSION THREAD

**Discussion by vidilab** [arendst/Tasmota#18284] [Q&A] [answered]
We have tried to program a ESP8685-WROOM-04-H4 chip with web installer and flashing with esptool, but although it seems successfull, it does not boot. Is this device type supported?
Based on this list [https://www.espressif.com/en/products/modules](https://www.espressif.com/en/products/modules), I think the chip is part of the ESP32-C3 Series, so was expecting it could be flashed with C3 Tasmota firmware.

**Comment by sfromis** [comment, marked as answer]
Yes, ESP8685 should be ESP32-C3 which Tasmota supports, while ESP32-C2 for the ESP8684 has no Arduino (and therefore Tasmota) support. Without exact details of your flashing and symptoms, no way of guessing what happens.

**Reply by barbudor** [reply]
Most ESP chips requires some proper configuration on some pins to properly boot
If you are using a bare module instead of a dev board, you may be missing that necessary environment to get a working hardware

**Comment by vidilab** [comment]
Not sure what was wrong originally, but we are now able to flash and run Tasmota on the chip.

**Comment by aj-way** [comment]
I've followed these instructions https://templates.blakadder.com/ESP8685-WROOM-06.html and using esptool.py v4.7.0 I've flashed with tasmota32c3.factory.bin without any problem. The "Hash of data verified" message makes me think the flash was successful.

But I can't get the device to boot. I've connected the pins as described; 3V3, GND, EN high, GPIO9 high. I was expecting a WiFi AP so I could configure it but I can't see an AP. I'm using a bench PSU so it's not due to lack of power.

Does anyone have any suggestions?

**Reply by sfromis** [reply]
What do you see on serial when trying to boot? 
Wiring gpio9 high should not be needed with the internal pullup present already.

**Comment by aj-way** [comment]
Hi sfromis,

Thanks for the reply. I've never tried making a serial connection, I always use esptool so I need to investigate how to doo that. It sounds like a useful diagnostic method.

Cheers.

**Reply by sfromis** [reply]
Since you used esptool, you already had a serial connection. Here's a couple of suggestions for terminal programs to follow what happens: https://tasmota.github.io/docs/Getting-Started/#__tabbed_2_3

If using the recommended Tasmota web installer, this tool also has a terminal feature built-in, without needing to download anything.

**Comment by aj-way** [comment]
That's useful information, I'll try termite. Thank you.

**Comment by aj-way** [comment]
Update....

Termite proved to be a very useful tool. I could see the device was in a reboot cycle. I re-flashed the device and now its connected to my WiFi.

@sfromis , thanks for the help.
